site/docs/foundations/data-visualization/chart-anatomy.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Chart anatomy
+layout: DetailTechnical
+sidebar:
+  priority: 4
+---
+
+Understanding the principles of chart anatomy is important to learning how the different parts of Salt charts work. 
+
+This will include: 
+
+- Axis title
+- Axis label
+- Legend
+- Grid lines
+
+<Image
+  src="/img/foundations/dataviz-anatomy.png"
+  srcDark="/img/foundations/dataviz-anatomy-dark.png"
+  alt="Diagram of a typical bar chart anatomy."
+/>
+
+## Titles
+
+The chart’s title should summarize the data on display. If revealing key findings, supporting an argument, story or message, capture it in the title.
+
+## Axes
+
+Horizontal (x) and vertical (y) axes display two variables and include their range, scale and units of measurement. Axes are used in two-dimensional charts, with each axis representing a measure of data.
+
+### Axes title
+
+The axis title should summarize the variables shown. Consider including the unit of measurement in the title to avoid repeating it on the axis labels, e.g., M for millions, or for percentages.
+
+### Axes labels
+
+Label each axis clearly and concisely to reduce visual clutter. Simplify axis labels by including the unit of measurement in the axis title. Keep labels horizontal where possible to maintain readability.
```

Comment:
You're repeating yourself in this sentence. See Axes title.

Reply:
I think there is a point missing here with horizontal labels? The goal is to keep labels horizontal 
(not vertical or at an angle) to improve legibility... which may require changing the orientation of the chart so horizontal labels align with the chart graphics.